### DISCUSSION TITLE
Pressure Boundary Condition set by MultiApp

### DISCUSSION THREAD

**Discussion by makeclean** [idaholab/moose#27139] [Q&A General]
Hi, I have a THM multiapp which calculates pressure, I have transferred pressure back to the main application as an `AuxVariable`, I would like to set the `Pressure` boundary condition using the data from the AuxVariable on the sideset. What is the right way to for me to set the data in the Pressure boundary condition?

**Comment by GiudGiud** [comment]
Hello

> I would like to set the Pressure boundary condition using the data from the AuxVariable on the sideset

which solve is this boundary condition for? A multiD flow solver or another THM one?
Or is this litteraly `Pressure` from the `Pressure` action for solid mechanics?

Guillaume

**Reply by makeclean** [reply]
I couldn't have been more vague could I? Its the solid mechanics pressure, https://mooseframework.inl.gov/source/bcs/Pressure.html

**Reply by GiudGiud** [reply]
ok Pressure only takes Real and Functions. @bwspenc @somu15 do you know which BC does pressure with auxvariables?
I m wondering if we just use this one but they will know
https://mooseframework.inl.gov/source/bcs/CoupledVarNeumannBC.html
